package.json

```diff
@@ -25,23 +25,40 @@
     "immutable": "3.7.6"
   },
   "devDependencies": {
-    "babel-cli": "^6.8.0",
-    "babel-core": "^6.8.0",
-    "babel-eslint": "^6.0.4",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-transform-runtime": "^6.8.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-stage-0": "^6.5.0",
-    "chai": "^3.2.0",
-    "eslint": "^1.4.1",
-    "eslint-config-airbnb": "0.0.8",
-    "mocha": "^2.3.2",
-    "sinon": "^1.15.4"
+    "babel-cli": "^6.24.1",
+    "babel-core": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "chai": "^3.5.0",
+    "mocha": "^3.2.0",
+    "semistandard": "^11.0.0",
+    "sinon": "^2.1.0",
+    "snazzy": "^7.0.0"
   },
   "scripts": {
-    "lint": "eslint .",
+    "lint": "semistandard | snazzy",
     "compile": "babel src --out-dir dist",
-    "test": "NODE_ENV=test mocha --compilers js:babel-core/register",
-    "prepublish": "npm run compile"
+    "test": "npm run lint && mocha --compilers js:babel-core/register",
+    "prepublish": "npm run test && npm run compile"
```

Comment:
The addition of `npm run test` due to https://github.com/redbadger/immutable-cursor/commit/c8603d24a8e6f178be330c43e9ec9c79fe9b6f39